src/person/person.go

```diff
@@ -71,3 +76,10 @@ func (p *Person) CanBeGE() bool {
 	}
 	return false
 }
+
+// IsBannerID returns true if id is a 95 number
+func IsBannerID(stringID string) bool {
+	clean := strings.Replace(stringID, "-", "", -1)
+	re := regexp.MustCompile("95[0-9]{7}")
```

Comment:
Sorry, I missed this before: we should require this to be the entire string, not just match somewhere in the string.  e.g., `^95[0-9]{7}$`.  Also, you can use `\d` in place of `[0-9]`, but I don't really care either way.

Reply:
This is far too minor to hold up merging, so I'll file a new ticket